packages/driver/cypress/e2e/util/what-is-circular.cy.ts

```diff
@@ -0,0 +1,192 @@
+import { whatIsCircular } from '../../../src/util/what-is-circular'
```

Comment:
Previous test here: https://github.com/cypress-io/what-is-circular/blob/master/test.js